kartothek/io_components/merge.py

```diff
@@ -1,8 +1,18 @@
 import logging
+from functools import partial, reduce
+from typing import TYPE_CHECKING, Callable, Dict, List, Optional, cast
+
+import pandas as pd
 
 from kartothek.core.dataset import DatasetMetadata
+from kartothek.core.factory import DatasetFactory
 from kartothek.io_components.metapartition import MetaPartition
-from kartothek.io_components.utils import _instantiate_store
+from kartothek.io_components.read import dispatch_metapartitions_from_factory
+from kartothek.io_components.utils import _instantiate_store, _make_callable
+
+if TYPE_CHECKING:
+    from simplekv import KeyValueStore
```

Comment:
```suggestion
    from simplekv import KeyValueStore  # noqa: F401
```

linting fails: `kartothek/io_components/merge.py:14:5: F401 'simplekv.KeyValueStore' imported but unused`